### ISSUE TITLE
NanoNVA keeps freezing

### ISSUE THREAD

**Issue by Double00Dalo** [NanoVNA-Saver/nanovna-saver#350]
**Describe the bug**
Whenever I click to connect to my nanovna, it freezes and stops responding
**To Reproduce**
1. Open nanovnasaver
2. Click on connect to device

**Expected behavior**
I expected it to continue normally and show graphs from my nanovna on the software.

**Screenshots**
Not exactly sure how to do this

**Desktop (please complete the following information):**
 - OS: [Windows 10}
 - NanoVNA-Saver version: [e.g. 0.3.8]

**Additional context**

**Comment by DL1ABR** [comment]
As astrocfi wrote and I have the same problem:
I just ran into the same problem with 0.3.8 and the NanoVNA-F. I removed the NanoVNASaver.ini file as suggested by @tczerwonka, but it didn't help. I'm running Windows 10.
Sometime Saver is running properly for some short time, then it quickly freezes - No Respond.
Restart Saver helps sometimes, sometimes not and Saver freezes at once after start.
Restart of PC helps sometimes, sometimes not and Saver freezes at once after start.
